### ISSUE TITLE
Could not find com.android.tools.build:gradle:3.0.0.

### ISSUE THREAD

**Issue by b95505017** [gradle/android-cache-fix-gradle-plugin#43]
After adding plugins block in my `app/build.gradle`:

```
plugins {
    id "org.gradle.android.cache-fix" version "0.3.4"
}
```

The log shows:

```
Could not find com.android.tools.build:gradle:3.0.0.
Searched in the following locations:
    https://plugins.gradle.org/m2/com/android/tools/build/gradle/3.0.0/gradle-3.0.0.pom
    https://plugins.gradle.org/m2/com/android/tools/build/gradle/3.0.0/gradle-3.0.0.jar
Required by:
    project :app > org.gradle.android.cache-fix:org.gradle.android.cache-fix.gradle.plugin:0.3.4 > gradle.plugin.org.gradle.android:android-cache-fix-gradle-plugin:0.3.4
```

**Comment by artem-zinnatullin** [comment]
Most likely you're using pre 3.x version of Android Gradle Plugin, which
you should update to 3.x if you want to use this plugin.

Secondly, Android Gradle Plugin is published to Google's maven repo which
you need to add to repositories section.

Please carefully read and follow instructions on upgrading to Android
Gradle Plugin 3.x before applying thus plugin
https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html

**Comment by b95505017** [comment]
Oh sorry, I forgot to mention. I'm using `com.android.tools.build:gradle:3.1.0-beta3`

**Comment by lptr** [comment]
I think you need to use the `buildscript {}` notation to add the plugin, and you need to add the `google()` repository:

```gradle
buildscript {
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath "org.gradle.android:android-cache-fix-gradle-plugin:0.3.4"
    }
}
```

**Comment by b95505017** [comment]
@lptr It doesn't work.

```
Could not find org.gradle.android:android-cache-fix-gradle-plugin:0.3.4.
Searched in the following locations:
    https://dl.google.com/dl/android/maven2/org/gradle/android/android-cache-fix-gradle-plugin/0.3.4/android-cache-fix-gradle-plugin-0.3.4.pom
    https://dl.google.com/dl/android/maven2/org/gradle/android/android-cache-fix-gradle-plugin/0.3.4/android-cache-fix-gradle-plugin-0.3.4.jar
    https://jcenter.bintray.com/org/gradle/android/android-cache-fix-gradle-plugin/0.3.4/android-cache-fix-gradle-plugin-0.3.4.pom
    https://jcenter.bintray.com/org/gradle/android/android-cache-fix-gradle-plugin/0.3.4/android-cache-fix-gradle-plugin-0.3.4.jar
Required by:
    project :
```

**Comment by lptr** [comment]
Sorry, you also need `gradlePluginPortal()`:

```gradle
buildscript {
    repositories {
        google()
        jcenter()
        gradlePluginPortal()
    }
    dependencies {
        classpath "org.gradle.android:android-cache-fix-gradle-plugin:0.3.4"
    }
}
```
